src/gui/SettingsWidgetGeneral.ui

```diff
@@ -106,6 +106,13 @@
         </widget>
        </item>
        <item>
+        <widget class="QCheckBox" name="ignoreGroupExpansionCheckBox">
+         <property name="text">
+          <string>Do not mark a database as modified when groups are expanded</string>
```

Comment:
Do we want to expand this definition to be "Don't mark database as modified for non-data changes (e.g., expanding groups)"? This setting could apply to other things that we implement such as sort order of columns, visible columns, etc.